development/views_api.py

```diff
@@ -39,7 +39,7 @@ def convert_data(req_data):
     """ Recieve a dictionary and return a new dictionary. """
```

Comment:
This is very vague, I know it is not part of this PR but you can specify a little more about what is going on here

Reply:
done!